packages/app/src/providers/h5grove/__snapshots__/h5grove-api.test.ts.snap

```diff
@@ -670,13 +670,13 @@ exports[`test file matches snapshot 1`] = `
       "endianness": "little-endian",
       "size": 128,
     },
-    "value": [
+    "value": Float64Array [
       0,
       1,
       2,
       3,
       4,
-      null,
+      Infinity,
```

Comment:
The maximum finite float128 value gets converted to float64 `Infinity`. I would have expected Numpy to clamp it to the maximum finite float64 value instead. :thinking: 

Now that I think of it, a similar thing happens with the smallest float128 value greater than 0 (the value of the `float128_scalar` dataset in the sample file) => h5grove sends 0 in float64 — not the smallest float64 value greater than 0, as I would have expected.

Reply:
> The maximum finite float128 value gets converted to float64 Infinity

Sounds like a safe behaviour. It would be weird to convert a finite value to another while you know it's not the same value.

Reply:
> It would be weird to convert a finite value to another while you know it's not the same value.

That's exactly what happens when you cast an int64 to int32, though. Obviously `inf` cannot be represented in int32, but still, in float, it's a special value... The rounding to 0 makes more sense in that regard.

I'm just asking in case the code in h5grove is wrong somehow — maybe the float128 are cast by Python before numpy can convert them properly, or something of the sort. :shrug: